src/Tests/Service/U2fServiceRegistrationTest.php

```diff
@@ -20,11 +20,12 @@
 
 use Mockery as m;
 use PHPUnit_Framework_TestCase as TestCase;
-use Surfnet\StepupU2fBundle\Service\RegistrationService;
 use Surfnet\StepupU2fBundle\Service\RegistrationVerificationResult;
+use Surfnet\StepupU2fBundle\Service\U2fService;
+use Surfnet\StepupU2fBundle\Value\AppId;
 use u2flib_server\Error;
 
-final class RegistrationServiceTest extends TestCase
+final class U2fServiceRegistrationTest extends TestCase
```

Comment:
please add explanatory docblock about why the expectations for the U2F lib are added here as well.